internal/controller/postgrescluster/instance.go

```diff
@@ -1086,11 +1087,14 @@ func (r *Reconciler) reconcileInstance(
 	if err == nil {
 		postgresWALVolume, err = r.reconcilePostgresWALVolume(ctx, cluster, spec, instance, observed, clusterVolumes)
 	}
+	if err == nil {
+		tablespaceVolumes, err = r.reconcileTablespaceVolumes(ctx, cluster, spec, instance, clusterVolumes)
+	}
 	if err == nil {
 		postgres.InstancePod(
```

Comment:
:question: Should this function start taking a parameter struct to bring the number of arguments down?

Reply:
I'm not feeling that as a necessity at this moment, but would definitely consider it in the future